ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/util/MapperUtilTest.java

```diff
@@ -202,13 +209,24 @@ public void canContinueOnlineIfNoPaperResponse() {
     }
 
     @Test
-    public void cantContinueOnlineIfStaffUploadedDocumentPresent() {
-        CCDCase ccdCase =
-            SampleData.withPaperResponseFromStaffUploadedDoc();
-
-        YesNoOption result = hasPaperResponse.apply(ccdCase);
-        assertThat(result).isEqualTo(YesNoOption.YES);
-
+    public void cantContinueOnlineIfAnyPaperResponseDocumentPresent() {
```

Comment:
If you're willing to migrate this to JUnit 5 you can use a parameterized test here